runtime/android/core_internal/src/org/xwalk/core/internal/XWalkContent.java

```diff
@@ -132,9 +136,21 @@ private void setNativeContent(long newNativeContent) {
 
         assert mNativeContent == 0 && mCleanupReference == null && mContentViewCore == null;
 
+        boolean animated;
+        //If app developers want to assign the CompositingSurfaceType for a single xwalkview,
+        //"animatable" attribute of this namespace must be defined in activity.xml, while an attrs.xml
+        //that to define the type of "animatable" is also needed under res/values/
+        String namespace="http://schemas.android.com/apk/res-auto";
```

Comment:
Please add final, and move final String namespace="http://schemas.android.com/apk/res-auto";  to else